### PR TITLE
[SH4RE-28] fe of seach results

### PR DESCRIPTION
**Pull request by americobarros**

### CODE REVIEW

**Review by WasifButt — DISMISSED** [review]
LGTM

File: SH4RE/SH4RE.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hannahbrooks) [inline SH4RE/SH4RE.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: do we need this file?

File: SH4RE/SH4RE.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -110,4 +110,4 @@
     }
```
Comment (hannahbrooks) [inline SH4RE/SH4RE.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:110]: do we need this file as well?

File: SH4RE/SH4RE/Search/Components/ProductCard.swift [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+struct ProductCard: View {
+    var listing: Listing;
+    var image:UIImage
+    
+    let screenSize: CGRect = UIScreen.main.bounds
+    
+    var body: some View {
+        
+        ZStack (alignment: .bottom){
```
Comment (hannahbrooks) [inline SH4RE/SH4RE/Search/Components/ProductCard.swift]: is this needed? or is this to keep them above the navbar

**Review by hannahbrooks — DISMISSED** [review]

File: SH4RE/SH4RE.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 2 above)
Comment (americobarros) [reply SH4RE/SH4RE.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:110]: yes its the packages and their versions

File: SH4RE/SH4RE/Search/Components/ProductCard.swift (hunk 3 above)
Comment (americobarros) [reply SH4RE/SH4RE/Search/Components/ProductCard.swift]: the nice youtube lady told me to do it but ur right I don't think it does anything
I think it mayyyy be for then I add the favorite button tho. Speaking of which, where are we even gonna store favorited listings in the app?

File: SH4RE/SH4RE.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (hunk 1 above)
Comment (americobarros) [reply SH4RE/SH4RE.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings]: nah i dont think so

**Review by hannahbrooks — APPROVED** [review]

**Pull request merged**